### ISSUE TITLE
Abnormal KCO

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6496]
**Preferred term label:** 
Abnormal KCO
**Synonyms**
Diffusion capacity of the lung for carbon monoxide (CO) rate constant
**Definition (free text, please give PubMed ID)**
The KCO is a measurement of the rate constant for alveolar uptake of CO during breath-holding in the single breath measurement of DLCO at full inflation. The DLCO is derived as the product of the KCO and the single breath alveolar volume (VA) divided by PB-PH2O. PMID: 11334115
**Parent term (use hpo.jax.org/app)**
Abnormality on pulmonary function testing HP:0030878
**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Combined pulmonary fibrosis-emphysema syndrome ORPHA:300564
Idiopathic pulmonary fibrosis ORPHA:2032 
Sarcoidosis ORPHA:797 
Asbestosis ORPHA:2302
Vasculitis ORPHA:52759
Chronic thromboembolic pulmonary hypertension ORPHA:70591 
Pulmonary arteriovenous malformations ORPHA:2038
Congenital lobar emphysema ORPHA:1928
Genetic neuromuscular disease ORPHA:183497
Idiopathic pulmonary haemosiderosis ORPHA:99931
Diffuse alveolar haemorrhage ORPHA:90060
**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added term
